lib/node/nodes/georeference-city.js

```diff
@@ -7,7 +7,8 @@ var PARAMS = {
     source: Node.PARAM.NODE(Node.GEOMETRY.ANY),
     city: Node.PARAM.STRING(),
     admin_region: Node.PARAM.STRING(),
-    country: Node.PARAM.STRING()
+    country: Node.PARAM.STRING(),
+    provider: Node.PARAM.NULLABLE(Node.PARAM.STRING(), 'mapzen')
```

Comment:
Who is going to provide this param?. Should the user choose what 3rd party service we use internally?